deptofdefense/move.mil
# More favicons

Addresses #205.

## Checklist

I have…


## Summary of Changes

- Uses a simplified version of the USTRANSCOM logo with stronger lines and no text for the favicon files at 32x32 and below.
- Adds Android Chrome icons.
- Adds MS tile icons (generated by realfavicongenerator.net)
- Adds safari mask SVG, used for pinned tabs and the touch bar
- Adds iOS icon.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `more-favicons`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in your Web browser of choice.

## Screenshots

### favicon in Chrome tab before

<img width="85" alt="screen shot 2017-12-01 at 12 16 22 pm" src="https://user-images.githubusercontent.com/26554826/33495866-1a4e56c2-d696-11e7-8f13-715fc3a82d37.png">

### favicon in Chrome tab after

<img width="85" alt="screen shot 2017-12-01 at 12 26 45 pm" src="https://user-images.githubusercontent.com/26554826/33495878-218b085e-d696-11e7-8c43-2e200071296e.png">

### Safari

<img width="231" alt="screen shot 2017-12-01 at 12 39 28 pm" src="https://user-images.githubusercontent.com/26554826/33495892-2a8b6048-d696-11e7-8fb4-a3aee0e6cb41.png">
